### PR TITLE
Changelog howto

### PR DESCRIPTION
**Pull request by rnelson0**

### CODE REVIEW

File: _posts/2017-01-13-github_changelog_generator-usage.md [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+* [4.1.0](https://github.com/voxpupuli/puppet-r10k/blob/c61493173a6ba531d00aa2d873bc3feb1b0069d2/CHANGELOG.md): First changelog generated by tooling. No matter how far back we scroll, we see release numbers, dates, and links to the release tags. Underneath each release header is a link to the full diff between the tags, followed by the list of closed issues and merged PRs, if any, and a link to each and the author of PRs. Each entry's description matches the title. This has consistency AND accuracy!
+
+To generate a changelog for a Vox Pupuli module, all you need to do is run a single command at the top level of the repo:
+```
+bundle exec rake changelog
+```
+That's it! After generating your changelog, review it (I advise committing your change and viewing it in your branch on GitHub), and make sure it looks okay. Update the title or label on Issues and PRs and generate the change log again. Review and tweak as necessary until the changelog is accurate and the noise is removed. Commit again and attach it to your release PR.
```
Comment (bastelfreak) [inline _posts/2017-01-13-github_changelog_generator-usage.md]: "change log" should be "changelog"?

File: _posts/2017-01-13-github_changelog_generator-usage.md [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+title: Using github_changelog_generator
+date: 2017-01-13
+github_username: rnelson0
+twitter_username: rnelson0
+---
+
+With the upgrade to Puppet 4 in all of our modules, we have also added the gem [github_changelog_generator](https://github.com/skywinder/github-changelog-generator). This is an automated tool to generate a `CHANGELOG.md` based on the Issues and PRs from a github repo with minimal manual intervention. No more forgetting about an important change or describing it wrong, the gem handles that for you.
+
+The changelog contents are based on a few things:
+* The current version of the Puppet module per `metadata.json`
```
Comment (bastelfreak) [inline _posts/2017-01-13-github_changelog_generator-usage.md]: should we edit the version in `metadata.json` before or after the generator parsed it?

**Comment by bastelfreak** [comment]
thanks for the guide. We should probably link this blogpost and mine at this page? https://voxpupuli.org/docs/#releasing-a-new-version-of-a-module

**Comment by rnelson0** [comment]
Yes, I think so.

**Pull request merged**